data/remnant/remnant 1 introduction.txt

```diff
@@ -2435,6 +2435,60 @@ mission "Remnant: Salvage 6"
 
 
 
+mission "Remnant: Scanning Tolerances"
+	name "Deliver Data to <planet>"
+	description `Deliver data crystal to the researchers on <destination>.`
+	landing
+	deadline 15
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+	to offer
+		has "license: Remnant Capital"
+		not "remnant untrusted"
+		not "Deep: Remnant 3: Revealed: done"
+		"reputation: Remnant" > 1
+		random < 30
+	on offer
+		conversation
+			`You were just in the process of disembarking from <ship> when a young Remnant jogs up to you. "Good day, Captain <first>. Sorry to interrupt, but we have an urgent delivery that needs to go to Torza on <destination>. Could you deliver this please?"`
+			choice
+				`	"Yes, I can."`
+					goto agreement
+				`	"What is it?"`
+					goto questioning
+				`	"Sorry, not right now."`
+					defer
+			label questioning
+			`	"A data archive with the latest briefings and analysis. Heavily shielded and encrypted, of course, so no need to worry about it being accidentally showing up on a scanner. It does have to get there by <date> So, could you do it?" They hold out a small case with a data crystal visible through the window.`
+			choice
+				`	"Yes, I can."`
+					goto agreement
+				`	"Sorry, not right now."`
+					defer
+			label agreement
+			`	"Thank you." They hand you the case. "Take it to <destination> by <date>. Torza will meet you there."`
+	on complete
+		event "remnant: scanning tolerance"
+		conversation
+			`You step out of the <ship> to see a non-descript Remnant standing attentively next to the landing pad. "Captain <first>! We have been waiting for you." They lead you into a nearby bunker, inside a single Remnant is standing over what appears to be a hologram display of fortifications and what appears to be some troops and ships moving around.`
```

Comment:
Am I sure it's a single Remnant and not just what appears to be a single Remnant? Is it really a bunker or does it just appear to be one?
```suggestion
			`You step out of the <ship> to see a non-descript Remnant standing attentively next to the landing pad. "Captain <first>! We have been waiting for you." They lead you into a nearby bunker, inside a single Remnant is standing over a hologram displaying fortifications and what appear to be troops and ships moving around.`
```